utils/fixed_decimal/src/decimal.rs

```diff
@@ -2078,6 +2078,185 @@ impl FixedDecimal {
     }
 }
 
+/// A struct containing a [`FixedDecimal`] significand together with an exponent, representing a
+/// number written in scientific notation, such as 1.729×10³.
+/// This structure represents any 0s shown in the significand and exponent,
+/// and an optional sign for both the significand and the exponent.
+#[derive(Debug, Clone, PartialEq)]
+pub struct ScientificDecimal {
```

Comment:
Please put these in their own source files (src/scientific.rs and src/compact.rs), and then re-export them from the top level.